### ISSUE TITLE
Add some code examples. 

### ISSUE THREAD

**Issue by bowenjfung** [bowenjfung/tuesdayteardown#2]
I'm pretty sure we've run through some that aren't sensitive.

**Comment by danmyles** [comment]
Agreed — would be nice to have a shared codebase for tips and tricks. If I have time I'll create some dummy survey data (rather than my own data) for my purrr session that can be uploaded here. Another long term goal might be to make our own package with helper functions etc that we each have written over time?

**Comment by danmyles** [comment]
Also, would suggest that this is more or less the same as the other "issue"?

**Comment by niki-chen** [comment]
I got a bunch of automated reports and stuff using Rmd if anyone is
interested in some boilerplate code... it's boring as hell but it gets the
job done.

**Comment by danmyles** [comment]
I've also got some nice custom css for R Markdown. It's just a hack of the tufte theme to fix up a few things I didn't like about it. I'd be happy to upload the css + boiler plate for that too.
